### PR TITLE
Different TextAlignment and Padding for the items in the dropdown

### PR DESCRIPTION
**Pull request by AWhiteSheep**
The logic seems okay on my end.

### CODE REVIEW

**Comment by AWhiteSheep** [comment]
#218 #220

File: drop-down.android.ts [hunk 1]
```diff
@@ -409,13 +445,22 @@ function initializeDropDownAdapter() {
                     label.style.placeholderColor = owner.style.placeholderColor;
                 }
                 label.style.textDecoration = owner.style.textDecoration;
-                label.style.textAlignment = owner.style.textAlignment;
+                if (owner.nativeView.itemsTextAlignment !== itemsTextAlignmentProperty.defaultValue 
+                    && realizedViewType === 1) {
+                     label.style.textAlignment = owner.nativeView.itemsTextAlignment;
+                } else {
+                    label.style.textAlignment = owner.style.textAlignment;
+                }
```
Comment (PeterStaev) [inline drop-down.android.ts]: Prefer ternary operator here.

File: drop-down.android.ts [hunk 2]
```diff
@@ -409,13 +445,22 @@ function initializeDropDownAdapter() {
...
                 if (owner.style.fontSize) {
                     label.style.fontSize = owner.style.fontSize;
                 }
                 view.style.backgroundColor = owner.style.backgroundColor;
-                view.style.padding = owner.style.padding;
+                if (owner.nativeView.itemsPadding !== itemsPaddingProperty.defaultValue && realizedViewType === 1) {
+                    view.style["padding"] = owner.nativeView.itemsPadding;
+                } else {
+                    view.style.padding = owner.style.padding;
+                }
```
Comment (PeterStaev) [inline drop-down.android.ts]: Prefer ternary operator here.

File: drop-down.ios.ts [hunk 3]
```diff
@@ -353,17 +371,49 @@ class DropDownListPickerDelegateImpl extends NSObject implements UIPickerViewDel
...
+                itemsPaddingRight = itemsPadding[1];
+                itemsPaddingBottom = itemsPadding[2];
+                itemsPaddingLeft = itemsPadding[1];
+            } else if (itemsPadding.length === 4) {
+                itemsPaddingTop = itemsPadding[0];
+                itemsPaddingRight = itemsPadding[1];
+                itemsPaddingBottom = itemsPadding[2];
+                itemsPaddingLeft = itemsPadding[3];
+            }
+        }
```
Comment (PeterStaev) [inline drop-down.ios.ts]: This whole logic will have to be refactored to accommodate couple of things:
1. In NS it is supported to enter padding separated by `,`. So a similar function to https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/styling/style-properties.ts#L379 should be used
2. Paddings are not only numbers, they can also be `px`, etc. So you cant use `parseInt`. You must use a similar approach to https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/styling/style-properties.ts#L448 and use `Length.parse` to get the actual value.

File: drop-down.ios.ts [hunk 4]
```diff
@@ -353,17 +371,49 @@ class DropDownListPickerDelegateImpl extends NSObject implements UIPickerViewDel
...
 
         label.padding = {
-            top: utils.layout.toDeviceIndependentPixels(owner.effectivePaddingTop),
-            right: utils.layout.toDeviceIndependentPixels(owner.effectivePaddingRight),
-            bottom: utils.layout.toDeviceIndependentPixels(owner.effectivePaddingBottom),
-            left: utils.layout.toDeviceIndependentPixels(owner.effectivePaddingLeft)
+            top: utils.layout.toDeviceIndependentPixels(itemsPaddingTop),
+            right: utils.layout.toDeviceIndependentPixels(itemsPaddingRight),
+            bottom: utils.layout.toDeviceIndependentPixels(itemsPaddingBottom),
+            left: utils.layout.toDeviceIndependentPixels(itemsPaddingLeft)
```
Comment (PeterStaev) [inline drop-down.ios.ts]: Once `itemsPadding...` are a `Length` type, you can simply use here `itemsPaddingTop.toDevicePixels()` for example.

File: drop-down.ios.ts [hunk 5]
```diff
@@ -353,17 +371,49 @@ class DropDownListPickerDelegateImpl extends NSObject implements UIPickerViewDel
...
         };
 
         label.font = style.fontInternal.getUIFont(label.font);
-
-        switch (style.textAlignment) {
+        let itemsTextAlignment = owner.nativeView.itemsTextAlignment;
+        if (itemsTextAlignment === itemsTextAlignmentProperty.defaultValue) {
+            itemsTextAlignment = style.textAlignment;
+        }
```
Comment (PeterStaev) [inline drop-down.ios.ts]: Prefer `const` and ternary operator here.

**Comment by AWhiteSheep** [comment]
@PeterStaev it should work in practice would you review my changes?

**Comment by AWhiteSheep** [comment]
😃

**Pull request merged**